static/js/popovers.js

```diff
@@ -510,6 +510,51 @@ exports.register_click_handlers = function () {
         e.stopPropagation();
         e.preventDefault();
     });
+    $('body').on('click', '.view_edit_history', function (e) {
+        var msgid = $(e.currentTarget).data('msgid');
+        var row = current_msg_list.get_row(msgid);
+        var message = current_msg_list.get(rows.id(row));
+
+        popovers.hide_actions_popover();
+        $('#message-edit-history').modal("show");
+        channel.get({
+            url: "/json/messages/" + message.id + "/history",
+            data: {"message_id": JSON.stringify(message.id)},
+            success: function (data) {
+                var message_history = data.message_history;
+                _.each(message_history, function (msg) {
```

Comment:
So, if user has posted a message say "Hello" and later edits it to "Hello, all". In `message_history`, object that contains "Hello" has timestamp when it was edited to "Hello, all". So, I am changing timestamp such that "Hello, All" contains the timestamp when message was first edited.